src/main/kotlin/me/owdding/lib/waypoints/MeowddingWaypointHandler.kt

```diff
@@ -25,6 +25,9 @@ object MeowddingWaypointHandler {
     private var _waypoints: MutableList<MeowddingWaypoint> = mutableListOf()
     val waypoints: List<MeowddingWaypoint> get() = _waypoints
 
+    fun getWaypointsWithAnyTags(vararg tags: MeowddingWaypointTag): List<MeowddingWaypoint> = _waypoints.filter { it.tags.any { tag -> tags.contains(tag) } }
+    fun getWaypointsWithAllTags(vararg tags: MeowddingWaypointTag): List<MeowddingWaypoint> = _waypoints.filter { it.tags.containsAll(tags.toList()) }
```

Comment:
move the toList outside of the filter, rn it will create a new list for each waypoint